terraform/main.tf

```diff
@@ -185,10 +188,12 @@ module "production" {
     MAT_API_BASE_URL = "/production/manage-a-tenancy-api"
 
     WORKTRAY_URL       = "https://hlbctrial.outsystemsenterprise.com/manageatenancy/OfficerDashboard.aspx"
+    TENANCY_URL        = "https://hlbctrial.outsystemsenterprise.com/manageatenancy/CustomerDashboard.aspx"
     DIVERSITY_FORM_URL = "https://docs.google.com/forms/d/e/1FAIpQLScDI85GMCFl8c02DYGpf_cOxsjD83FNbNFEIWKs4u_HOydhKA/viewform"
+
```

Comment:
```suggestion
```

Reply:
There's a formatter for Terraform (`terraform fmt`) and a VS Code plugin for it.